Source/WebKit/Shared/Cocoa/SandboxExtensionCocoa.mm

```diff
@@ -175,16 +178,84 @@ String resolvePathForSandboxExtension(StringView path)
     return resolvedPath;
 }
 
+static ASCIILiteral sandboxExtensionType(SandboxExtensionType type)
+{
+    switch (type) {
+    case SandboxExtensionType::ReadOnly:
+        return "ReadOnly"_s;
+    case SandboxExtensionType::ReadWrite:
+        return "ReadWrite"_s;
+    case SandboxExtensionType::Mach:
+        return "Mach"_s;
+    case SandboxExtensionType::IOKit:
+        return "IOKit"_s;
+    case SandboxExtensionType::Generic:
+        return "Generic"_s;
+    case SandboxExtensionType::ReadByProcess:
+        return "ReadByProcess"_s;
+    }
+}
+
+static void logDiagnosticsInfo(StringView path)
+{
+    int ret = sandbox_check(getpid(), "file-read-data", static_cast<enum sandbox_filter_type>(SANDBOX_FILTER_PATH), path.utf8().data());
+    if (ret)
+        RELEASE_LOG_ERROR(Sandbox, "No read access to '%s'", path.utf8().data());
```

Comment:
Should any of these be marked `%{private}s` or even `%{sensitive}s`? The file paths could reveal sensitive details in logging.